docs/development.md

```diff
@@ -109,18 +110,20 @@ cd docs
 sphinx-build -b html ./ ./generated/
 ```
 
-#### Docstrings
+#### Docstrings:
 Docstrings are used to generate the API reference. 
 PrimeQA Docstrings follow the Google convention. 
 See [Sec. 3.8](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) for more details.
-Such doctrings can be used to auto-generate code comments using Sphinx tool (Example: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html)
+Such doctrings can be used to auto-generate code comments using Sphinx tool ([Example](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html))
 
 If you are using PyCharm, this docstring style can be selected from: `File --> Settings --> Tools --> Python Integrated Tools`.
 
 #### Notebooks: 
 Each contribution should have an easy to use notebook that exemplifiess how to instantiate and use the functionality. 
-An example for MRC on Tydi is here: https://github.ibm.com/ai-foundation/PrimeQA/blob/master/notebooks/mrc/tydiqa.ipynb
+An example for MRC on Tydi is [here](https://github.com/primeqa/primeqa/blob/main/notebooks/mrc/tydiqa.ipynb)
 
-#### Documentation Pages
+#### Documentation Pages:
 Documentation pages (such as this one) are generated from a collection of Markdown files located
-in `docs`. It also contains a subfolder calles `img` which contains any (legally usable) images needed for documentation purposes. 
+in `docs`. It also contains a subfolder calles `_static/img` which contains any (legally usable) images needed for documentation purposes. 
+
```

Comment:
Could we document the requirement to have the sphinx related block in the README.md and what to do when a new README.md is added.
Is there an error message or warning to check if the block is missing.

Reply:
- I've added a documentation which explains what to do when a new pkg/readme is added, you can check here: https://github.com/primeqa/primeqa/tree/issue279-redesign-doc/docs#new-package-added-to-primeqa & a link here in development.md file -> https://github.com/primeqa/primeqa/blob/issue279-redesign-doc/docs/development.md#deploying-sphinx-docs-as-project-online
- if the block is missing, an warning/error msg will appear on github-action log